### DISCUSSION TITLE
Contract on LCD*LCD*LCD*LCD*MTD (or LC*LC*LC*LC*MT) ¿inconsistency?

### DISCUSSION THREAD

**Discussion by ElJefeDelDesierto** [FeynCalc/feyncalc#292] [Q&A]
Hello great FeynCalc team.

First of all I want to thank you for this amazing package, and for your constant support. The world wouldn't be the same without FeynCalc or FeynCalc team, for sure. My question is the following. I define two functions A1 and A2 as

```
A1[a1_, a2_, b1_, b2_, c1_, c2_, d1_, d2_, i1_, i2_, j1_, j2_, m1_, m2_, n1_, n2_] :=
LCD[a1, a2, b1, b2] LCD[c1, c2, d1, d2] LCD[i1, i2, j1, j2] LCD[m1, m2, n1, n2]
A2[a1_, a2_, b1_, b2_, c1_, c2_, d1_, d2_, i1_, i2_, j1_, j2_, m1_, m2_, n1_, n2_] :=
Contract[ LCD[a1, a2, b1, b2] LCD[c1, c2, d1, d2] LCD[i1, i2, j1, j2] LCD[m1, m2, n1, n2]]
```

The function A2 is the same as the function A1, just expressed in terms of the metric tensor by use of the identity shown in this [post ](https://github.com/orgs/FeynCalc/discussions/291). The equality of both functions can be proven by computing

`Contract[ A1[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2]] - A2[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2] // Simplify`

or 

`Contract[ ( A1[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2] - A2[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2] ) ] // Simplify`

both giving zero. Thus, we expect that any contraction of the indices in A1-A2 will give zero as well. If we contract a1 and c1

`Contract[ ( A1[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2] - A2[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2] ) MTD[a1, c1] ] // Simplify
`

we get zero, as we expected. On the other hand, if we contract a1 and i1 written in this way

`Contract[(A1[a1, a2, b1, b2, c1, c2, d1, d2, a1, i2, j1, j2, m1, m2, n1, n2] - A2[a1, a2, b1, b2, c1, c2, d1, d2, a1, i2, j1, j2, m1, m2, n1, n2])] // Simplify`

we also get zero; however, if we compute this last expression by writting the metric tensor explicitly 

`Contract[(A1b[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2] - A2b[a1, a2, b1, b2, c1, c2, d1, d2, i1, i2, j1, j2, m1, m2, n1, n2]) MTD[a1, i1]] // Simplify`

we don't get zero (at least not explicitly); what I get is the following long expression instead
![LCD LCD Contract](https://github.com/user-attachments/assets/e221a5e1-e942-4a04-9d3e-286ae1608bac)

The same happens if I work with LC and MT instead of LCD and MTD, respectively. As far as I know, the last two expressions are the same contraction and both should give zero, shouldn't they? If this is correct, is there any way (or command) to simplify the last expression in such a way that it also gives zero explicitly?

FeynCalc 10.0.0 (stable version).
Mathematica 14.0.0.0.
Windows 11.

Thanks in advance and congratulations for your fantastic work!

**Comment by vsht** [comment]
Most likely it's another manifestation of the Schouten identity, 
especially since you have multiple products of Levi-Civita's with
lots of open indices.

You can search for it in the old forum

https://feyncalc.github.io/OldForum/thread.html

In general, showing explicitly that two expressions vanish by Schouten 
is extremely difficult.



Am 21.12.24 um 03:41 schrieb ElJefeDelDesierto:
